README.md

```diff
@@ -8,8 +8,9 @@ to prepare a dish that incorporates a basket of mystery ingredients.
 
 ## Download
 To get mystery installed quickly, run this:
+
 ```
-wget -O - https://github.com/asilvr/mystery/blob/main/install.sh | bash
+url -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/asilvr/mystery/main/install.sh -o install.sh > /dev/null 2>&1; chmod +x install.sh; sudo ./install.sh;
```

Comment:
Looks like you forgot the `c` in `curl` but I tested this and it works as expected. Good work!

Reply:
oh copy paste

> Please fix the typo then we'll get this merged!

Fixed